table.go

```diff
@@ -85,6 +85,58 @@ func (rt *RoutingTable) Close() error {
 	return nil
 }
 
+// NPeersForCPL returns the number of peers we have for a given Cpl
+func (rt *RoutingTable) NPeersForCpl(cpl uint) int {
```

Comment:
can you get away without this function, and just call
`len(GetPeersForCpl)` instead?

Reply:
Have removed `GetPeersForCpl` as we don't need it anymore.

Reply:
We've removed `GetPeersForCpl ` as we don't need it anymore.